stations.csv

```diff
@@ -21590,7 +21590,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 23596;Erlangen Busbahnhof;erlangen-busbahnhof;;;49.5960354;11.0008411;;DE;Europe/Berlin;f;f;f;t;f;f;;;f;;f;8089292;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;エアランゲン;에를랑겐;;;Эрланген;;;埃尔朗根
 23597;Busa di Vigonza;busa-di-vigonza;8302595;;45.429943;11.951722;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;8303949;t;;f;8302595;t;S02595;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 23598;Orehoved St.;orehoved-st;8601811;;54.939071;11.867738;;DK;Europe/Copenhagen;f;f;f;t;f;f;;;f;;f;8601811;t;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-23599;Paris Porte Maillot;paris-porte-maillot;8789015;;48.880126;2.282485;4916;FR;Europe/Paris;f;f;f;f;f;f;;;f;;f;8789015;t;;f;@FRPARPPM;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;Parigi;;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
+23599;Paris Porte Maillot;paris-porte-maillot;8789015;;48.880126;2.282485;4916;FR;Europe/Paris;f;f;f;t;f;f;;;f;;f;8789015;t;;f;@FRPARPPM;t;2365;t;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;Parigi;;Parijs;Paříž;;Párizs;パリ;파리;Paryż;;Париж;;;巴黎
```

Comment:
What is the 2365?

Reply:
This is the Flixbus station Id

Reply:
We don't need it. We already have the city id. Our code will select the nearest station.